docs/api/graphql_queries.md

```diff
@@ -183,18 +207,50 @@ Response:
 
 ## Querying Locations
 
-!!! tip
+You can get the Location object from any Content item object by querying for `_location` or `_allLocations`.
+When you use `_location`, the API picks the Location to return based on the current SiteAccess, or returns the main one.
```

Comment:
Or (if my memory serves me right) the location that was specified if either `locationId` or `urlAlias` were used as arguments.